### PR TITLE
Basic Getting Started category standardization part 1

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -10,7 +10,7 @@ GEM
     execjs (2.7.0)
     ffi (1.9.18)
     forwardable-extended (2.6.0)
-    jekyll (3.4.1)
+    jekyll (3.4.3)
```
Comment (mlocher) [inline Gemfile.lock]: Would be great if we could keep updates to dependencies or the tech stack separate from content updates (easier to review, and I'd like to get staging builds for each update to the tech stack/layout/...)

File: _basic/getting-started/dependency-cache.md [hunk 2]
```diff
@@ -14,13 +14,15 @@ category: Getting Started
...
+## What Is The Dependency Cache?
 
 On all Codeship Basic projects, we automatically cache the dependency directories for several popular tools and frameworks to speed up future builds.
 
 This means that all packages saved in these directories at the end of a build will be automatically loaded on the build machine for future builds.
 
-Packages in the following directories will be automatically cached:
+## Using The Dependency Cache
+
+Any packages, up to 500mb, in the following directories at the end of your build will be automatically cached:
```
Comment (mlocher) [inline _basic/getting-started/dependency-cache.md]: for some reason the `$HOME/cache` folder is missing from that list :(

File: _basic/getting-started/deployment-pipelines.md [hunk 3]
```diff
@@ -34,23 +36,60 @@ On the next push to this branch (in this case 'master') the deployment will be t
 
 Now you can add additional deployments or configure another deployment pipeline.
 
-## Add additional deployment methods
+### Deployment Integrations
+
+As part of our deployment pipelines, Codeship provides turnkey deployment integrations for many common hosting providers, such as AWS ElasticBeanstalk, Heroku, Google App Engine and more.
```
Comment (mlocher) [inline _basic/getting-started/deployment-pipelines.md]: AWS _Elastic Beanstalk_ (with a space)

File: _basic/getting-started/deployment-pipelines.md [hunk 4]
```diff
@@ -34,23 +36,60 @@ On the next push to this branch (in this case 'master') the deployment will be t
...
+
+To use a deployment integration, just click on the logo of your provider after creating your pipeline and add your authentication and configuration information as required.
+
+![Create Deployment Branch]({{ site.baseurl }}/images/basic/deployment-integration.png)
+
+### Custom Script Deployments
+
+While Codeship does provide many helpful deployment integrations, you may find that you want to run your own commands or your own custom scripts as part of a deployment pipeline.
+
+You can use the [Script Deployment]({{ site.baseurl }}{% link _basic/continuous-deployment/deployment-with-custom-scripts.md %}) to run your custom deployment commands or to execute other tasks right after or before a deployment. These will run as part of a deployment pipeline exactly as any of our deployment integrations would, but will rely on your scripts to provide exit status codes of `0` or `1` to indicate that they have either passed or failed.
```
Comment (mlocher) [inline _basic/getting-started/deployment-pipelines.md]: We could mention that the exit status can be anything other than `0` for the build to fail. Not a blocker to the PR though.

File: _basic/getting-started/deployment-pipelines.md [hunk 5]
```diff
@@ -34,23 +36,60 @@ On the next push to this branch (in this case 'master') the deployment will be t
...
+
+Get more flexible deployment workflows with wildcard deployment pipelines that trigger if a branch starts with a certain prefix. Use one deployment configuration for multiple branches and automatically deploy your feature, release, QA, etc. branches to the corresponding environments.
+
+See our article on [deployment pipelines]({{ site.baseurl }}{% link _basic/getting-started/deployment-pipelines.md %}) for a general introduction on how to use deployments on Codeship.
+
+## Static Branch Deployment Pipelines
+
+If you choose _Branch is exactly_ the branch name will need to match exactly the branch that is currently being tested for the deployment to be triggered.
+
+## Wildcard Branch Deployment Pipelines
```
Comment (mlocher) [inline _basic/getting-started/deployment-pipelines.md]: What's the difference between this paragraph on wildcard deployments and the one two headings above it?

File: _basic/getting-started/dependency-cache.md (hunk 2 above)
Comment (ethangj) [inline _basic/getting-started/dependency-cache.md]: Addressed

**Pull request merged**